tests/integration-tests/tests/createami/test_createami.py

```diff
@@ -11,133 +11,211 @@
 # See the License for the specific language governing permissions and limitations under the License.
 
 import logging
-from os import environ
+import time
 
+import boto3
 import pytest
 from assertpy import assert_that
-from packaging import version
-from utils import run_command
+from cfn_stacks_factory import CfnStack
+from troposphere import Template, iam
+from utils import generate_stack_name, run_command
+
+from tests.common.utils import generate_random_string, get_installed_parallelcluster_version, retrieve_latest_ami
+
+
+def test_build_image(
+    region,
+    instance,
+    os,
+    pcluster_config_reader,
+    architecture,
+    s3_bucket_factory,
+    build_image_custom_resource,
+    build_image,
+):
+    """Test build image for given region and os"""
+    # Get base AMI
+    # remarkable AMIs are not available for ARM and ubuntu2004 yet
+    if os != "ubuntu2004":
+        base_ami = retrieve_latest_ami(region, os, ami_type="remarkable", architecture=architecture)
+    else:
+        base_ami = retrieve_latest_ami(region, os, architecture=architecture)
+
+    image_id = "-".join(["integ", "test", "build", "image", generate_random_string()])
```

Comment:
minor: an alternative is simply: `image_id = f"integ-test-build-image-{generate_random_string()}"`

Reply:
done